README.md

```diff
@@ -54,6 +54,38 @@ e.g.
 pipenv run python generate_requests.py requests.har requests/test_checkbox.json test_checkbox
 ```
 
+---
+
+## Deployment with [Helm](https://helm.sh/)
+
+To deploy this application with helm, you must have a kubernetes cluster already running and be logged into the cluster.
+
+Log in to the cluster using:
+
+```
+gcloud container clusters get-credentials survey-runner --region <region> --project <gcp_project_id>
+```
+
+You need to have Helm installed locally
+
+1. Install Helm with `brew install kubernetes-helm` and then run `helm init --client-only`
+
+2. Install Helm Tiller plugin for _Tillerless_ deploys `helm plugin install https://github.com/rimusz/helm-tiller`
+
+
+### Deploying the app
+
+To deploy the app to the cluster, run the following command:
+
+```
+helm tiller run \
+-    helm upgrade --install \
+-    runner-benchmark \
+-    k8s/helm \
+-    --set host=${HOST} \
+-    --set container.image=${DOCKER_REGISTRY}/eq-survey-runner-benchmark:${IMAGE_TAG}
+```
+
 ## Future Improvements
```

Comment:
As mentioned it worth adding to the readme about changing the locust opts variable and what that would mean etc.

Apart from this, looks good 👍